kpatch-build/kpatch-build

```diff
@@ -672,7 +688,9 @@ if [[ -n "$USERSRCDIR" ]]; then
 		warn "--archversion is incompatible with --sourcedir"
 		exit 1
 	fi
-	SRCDIR="$USERSRCDIR"
+
+	[[ -z "$OOT_MODULE" ]] && SRCDIR="$USERSRCDIR" || SRCDIR="$OOT_SOURCE"
```

Comment:
One last question: does `OOT_MODULE` still need to depend on `USERSRCDIR` after these changes?